src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPathContextProvider.cs

```diff
@@ -77,6 +64,21 @@ public sealed class VsPathContextProvider : IVsPathContextProvider2
             _solutionManager = solutionManager ?? throw new ArgumentNullException(nameof(solutionManager));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _getLockFileOrNullAsync = BuildIntegratedProjectUtility.GetLockFileOrNull;
+
+            _projectContext = new Lazy<INuGetProjectContext>(() => {
```

Comment:
because I could not pass `_settings.Value` in `GetClientPolicy(...)` if the creation of the lazy object was not inside of a method, therefore I passed it to the constructor.